rbac/management/relation_replicator/outbox_replicator.py

```diff
@@ -30,6 +32,10 @@
 class OutboxReplicator(RelationReplicator):
     """Replicates relations via the outbox table."""
 
+    def __init__(self, log: Optional["OutboxLog"] = None):
```

Comment:
Do you keep the definition of OutboxLog later on purpose?

Reply:
I had to check myself but yes! Thanks for asking. The OutboxLog defines the protocol for type-safety. Then it can accept either the OutboxWAL (which does the normal DB stuff) or the InMemoryLog which both implement the protocol.